examples/msgpack/src/tests.rs

```diff
@@ -0,0 +1,48 @@
+extern crate rmp_serde;
```

Comment:
No need to decode. We should know the bytes, right? Just check against the bytes.